### PR TITLE
social media link

### PR DESCRIPTION
**Pull request by brunolajoie**
update electricitymap.tmrow.co into .org

### CODE REVIEW

File: web/views/pages/index.ejs [hunk 1]
```diff
@@ -9,11 +9,11 @@ co2Sub = function(str) {
         <meta http-equiv="content-type" content="text/html; charset=UTF8" />
         <meta name="description" content="Electricity Map is a live visualization of where your electricity comes from and how much CO2 was emitted to produce it." />
         <meta property="og:description" content="Electricity Map is a live visualization of where your electricity comes from and how much CO2 was emitted to produce it." />
-        <meta property="og:image" content="https://cloud.githubusercontent.com/assets/1655848/20340757/5ada5cf6-abe3-11e6-97c4-e68929b8a135.png" />
+        <meta property="og:image" content="https://user-images.githubusercontent.com/22095643/32440984-c98653e2-c2f5-11e7-979a-0e94155eefb5.png" />
         <meta property="og:title" content="<%= __('misc.maintitle') %>" />
         <meta property="og:type" content="website" />
         <!-- It's import to keep the old og:url, else, likes will not accumulate properly -->
```
Comment (corradio) [inline web/views/pages/index.ejs]: I don't think you read this line :p
TLDR: We can't change.

**Comment by brunolajoie** [comment]
:D

*Bruno Lajoie*
+33 6 67 95 10 92
brunolajoie@gmail.com
